lib/cocoapods/installer/pod_source_installer.rb

```diff
@@ -77,9 +77,7 @@ def clean!
       #
       def lock_files!(file_accessors)
         return if local?
-        each_source_file(file_accessors) do |source_file|
-          FileUtils.chmod('u-w', source_file)
-        end
+        FileUtils.chmod('u-w', source_files(file_accessors))
```

Comment:
this is not going to be terribly different, since `chmod` loops on the list of files

Reply:
yeah I realized that afterwards :/

Reply:
it's fine, it's certainly more readable this way

Reply:
I considered actually running the `chmod` command since that does accept a file list